static/app/views/alerts/rules/issue/addIntegrationRow.tsx

```diff
@@ -0,0 +1,114 @@
+import {useCallback, useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import PluginIcon from 'sentry/plugins/components/pluginIcon';
+import {space} from 'sentry/styles/space';
+import type {IntegrationProvider, Project} from 'sentry/types';
+import type {Organization} from 'sentry/types/organization';
+import useApi from 'sentry/utils/useApi';
+import {AddIntegrationButton} from 'sentry/views/settings/organizationIntegrations/addIntegrationButton';
+
+type Props = {
+  closeModal: () => void;
+  organization: Organization;
+  project: Project;
+  providerKey: string;
+  setHasError: (boolean) => void;
+};
+
+function AddIntegrationRow({
+  providerKey,
+  organization,
+  project,
+  closeModal,
+  setHasError,
+}: Props) {
+  const [provider, setProvider] = useState<IntegrationProvider | null>(null);
+
+  const api = useApi();
+  const fetchData = useCallback(() => {
+    if (!providerKey) {
+      return Promise.resolve();
+    }
+
+    const endpoint = `/organizations/${organization.slug}/config/integrations/?provider_key=${providerKey}`;
+    return api
+      .requestPromise(endpoint)
+      .then(integrations => {
+        setProvider(integrations.providers[0]);
+      })
+      .catch(() => {
+        setHasError(true);
+      });
+  }, [providerKey, api, organization.slug, setHasError]);
+
+  useEffect(() => {
+    fetchData();
+  }, [fetchData]);
+
+  if (!provider) {
+    return null;
+  }
+
+  const {metadata} = provider;
+
+  const buttonProps = {
+    style: {margin: 0},
```

Comment:
Used inline styles since this Button is an imported Sentry component with other styling

Reply:
You should still be able to use styled components, just need to pass that component into the styled helper like this:
```
styled(Button)`
  margin: 0;
`;
```
That should do the trick (and can do the same thing for the integration button).

mixing inline styles with styled components can make things a little tricky because it mixes css selector priorities with classes / element styles. (there's also implications on what happens during the render cycle in browsers based on that - in the element styles are applied, then the class styles. since this is changing spacing it could cause reflows / layout thrashes)